README.md

```diff
@@ -633,6 +633,61 @@ root@redis-pvc-restore:/mnt/gluster# cat clone_data
 glusterfs csi clone test
 ```
 
+### Create a glusterfs lite storage class to use loopback bricks (RWX)
+
+```
+[root@localhost]# cat glusterfs-lite-storage-class.yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: glusterfs-lite-csi
+provisioner: org.gluster.glusterfs
+parameters:
+  provisionerType: "loop"
```

Comment:
IMO, `provisionerType` is bit vague or confusing one. What about this `brickType` ? Any thoughts ?

Reply:
I like `brickType`.

Reply:
We have already used brickType to identify it is arbiter brick or regular brick. Using brickType here will add confusion. How about using `brickProvisionerType`?